core/lib/src/request/tests.rs

```diff
@@ -47,3 +47,23 @@ fn test_multiple_headers_merge_into_one_from_hyp() {
     assert_headers!("friend" => ["alice"], "friend" => ["bob"], "friend" => ["carol"]);
     assert_headers!("friend" => ["alice"], "friend" => ["bob"], "enemy" => ["carol"]);
 }
+
+struct MyConfig {
+    user_val: String,
+}
+
+#[tokio::test]
```

Comment:
Other async tests use `#[rocket::async_test]` (which defaults to the single-threaded runtime instead of the multi-threaded runtime), which I would prefer for consistency with other tests.